manifests/4.9/cluster-kube-descheduler-operator.v4.9.0.clusterserviceversion.yaml

```diff
@@ -68,6 +71,7 @@ spec:
       In addition to the profiles, the following parameters can be configured:
 
       * `deschedulingIntervalSeconds` - Set the number of seconds between descheduler runs. A value of `0` in this field runs the descheduler once and exits.
+      * `profileCustomizations` - Allows certain profile parameters to be tweaked, such as PodLifetime
```

Comment:
* Needs a period at the end
* Should this be "PodLifeTime" (T capitalized) as in the strategy name, or should it be "podLifetime" as in the actual field in the YAML?

Reply:
Updated to match the yaml field, and point to the readme for more info